src/plugins/kibana/public/settings/sections/indices/_create.js

```diff
@@ -23,7 +23,7 @@ uiModules.get('apps/settings')
 
   // this and child scopes will write pattern vars here
   const index = $scope.index = {
-    name: 'logstash-*',
+    name: config.get('indexPatternPlaceholder', 'logstash-*'),
```

Comment:
I've added a default value to be "backward compatible" because today `logstash-*` is the default value when you open the indices page. If we leave this blank, the following exception is thrown:

![empty-index-name](https://cloud.githubusercontent.com/assets/333276/15993268/52d3582e-30e0-11e6-8706-a1166ffd29d8.png)

```
TypeError: Cannot read property 'then' of undefined
    at $scope.refreshFieldList (:5601/emy/bundles/kibana.bundle.js?v=8467:135045)
    at Object.fn (:5601/emy/bundles/commons.bundle.js?v=8467:62383)
    at Scope.$digest (:5601/emy/bundles/commons.bundle.js?v=8467:43059)
    at Scope.$apply (:5601/emy/bundles/commons.bundle.js?v=8467:43330)
    at done (:5601/emy/bundles/commons.bundle.js?v=8467:37779)
    at completeRequest (:5601/emy/bundles/commons.bundle.js?v=8467:37977)
    at XMLHttpRequest.requestLoaded (:5601/emy/bundles/commons.bundle.js?v=8467:37918)
```